src/inmanta/env.py

```diff
@@ -58,8 +59,19 @@ class PackageNotFound(Exception):
     pass
 
 
-class ConflictingRequirements(Exception):
-    pass
+class ConflictingRequirements(CompilerException):
+    def __init__(
+        self, msg: Optional[str] = None, conflicts: Optional[List[Tuple[Requirement, Optional[version.Version]]]] = None
+    ):
+        self.conflicts = conflicts
+        self.msg = msg
+
+    def get_msg(self) -> str:
+        message: str = self.msg if self.msg else ""
```

Comment:
```suggestion
        message: str = self.msg if self.msg else "Conflicting requirements:"
```
Otherwise the `\n\t` might look weird.